src/App.css

```
@@ -1,33 +1,26 @@
 .App {
   text-align: center;
 }
-
+body {background-color: #282c34}
```

It could be in the same code style 
```suggestion
body {
  background-color: #282c34;
}
```

---

fixed